e2e/run_test.go

```diff
@@ -46,10 +46,14 @@ func TestRunFromStdin(t *testing.T) {
 func TestRunFromStdinIssue309(t *testing.T) {
 	t.Parallel()
 
-	bucket := s3BucketFromTestName(t)
+	// This test fails with gcs as it accepts percent encoding
+	// for `X-Amz-Copy-Source` and does not respect `+` as `space character`.
+	// skip it as it is not planned to write a workaround for gcs.
+	skipThisIfGoogleEndpoint(t)
```

Comment:
Let's change the function signature to accept a string. The implementation could leverage `t.Skipf` so that we can see why certain tests are skipped instead of checking the source code for comments.